### PR TITLE
Added servicebus resource

### PR DESCRIPTION
**Pull request by Knyatsine**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -7,8 +7,9 @@ trigger:
 pr: none
 
 variables:
-  buildConfiguration: 'release'
-  buildPlatform: 'anycpu'
+  - group: Prod Management Resources
```
Comment (nbowes24) [inline azure-pipelines.yml:10]: Is this variable group needed for the build?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -51,7 +52,8 @@ jobs:
       projects: 'src/**/*.UnitTests.csproj'
       arguments: '--configuration $(buildConfiguration) --no-build'
 
-  - template: azure-pipelines-templates/dependency-check.yml@das-platform-building-blocks
+  - template: azure-pipelines-templates/build/step/app-build.yml@das-platform-building-blocks
```
Comment (nbowes24) [inline azure-pipelines.yml:55]: Would be good to make use of a pipeline-templates folder as seen in other apprentice-commitments repos

File: azure-pipelines.yml (hunk 1 above)
Comment (adam230594) [reply azure-pipelines.yml:10]: Its needed for the dependency check variables

File: azure/template.json [hunk 3]
```diff
@@ -202,6 +205,33 @@
...
+            "resourceGroup": "[parameters('sharedEnvResourceGroup')]",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "[concat(variables('deploymentUrlBase'),'role-assignments/role-assignment-service-bus.json')]",
+                    "contentVersion": "1.0.0.0"
+                },
+                "parameters": {
+                    "principalId": {
+                        "value": "[reference('app-service').outputs.managedServiceIdentityId.value]"
```
Comment (adam230594) [inline azure/template.json:221]: You're going to want the staging slot managedserviceidentity as well

**Review by cpwevans — APPROVED** [review]

**Pull request merged**